.github/workflows/fossa.yml

```diff
@@ -9,19 +9,14 @@ jobs:
   build:
     runs-on: ubuntu-latest
     # build-tools is built from ../../tools/build-tools.Dockerfile
```

Comment:
If we don't use the build-tools here, the comment can be deleted

Reply:
:+1: